src/docs/guides/laserfiche-tables/index.md

```diff
@@ -0,0 +1,4 @@
+<!--Copyright (c) Laserfiche.
+Licensed under the MIT License. See LICENSE in the project root for license information.-->

```

Comment:
Replace "Licensed under the MIT License. See LICENSE in the project root for license information." with 
"See LICENSE and LICENSE-CODE in the project root for license information."